CHANGELOG.md

```diff
@@ -35,7 +35,9 @@ These changes likely break some apps, please read the
   - `widget.AccordionContainer` (now `widget.Accordion`)
   - `layout.NewFixedGridLayout()` (now `layout.NewGridWrapLayout()`)
   - `widget.ScrollContainer` (now `container.Scroll`)
-  - `widget.SplitContainer` (now `containerr.Spilt`)
+  - `widget.SplitContainer` (now `container.Spilt`)
+  - `widget.Group` (replaced by `container.Card`)
```

Comment:
It’s widget.Card, isn’t it?

Reply:
oops :)

Fixed the other comment with a new line too - thanks